.github/workflows/ci.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     strategy:
       matrix:
         # Always keep a window of "5 most recent" including (or on top with) HEAD
-        ruby: ["2.5.9", "2.6.8", "2.7.4", "3.0.2", "latest"]
+        ruby: ["2.6.8", "2.7.4", "3.0.2", "3.1.1", "latest"]
```

Comment:
Can you please allow failures (see `exclude` section) for the new ruby version and remove `2.5.9` from that list.